src/loaders/configure/configure.js

```diff
@@ -0,0 +1,37 @@
+import { setAPI } from '../api/api'
```

Comment:
This is similar to the former "configure" file, except that it can output the interface, so it can be stored to the class who calls it for NPM..  It can also set the window, as is seen with the CDN builds

example --
nr = new Agent()
nr.noticeError() ---> (instead of window.newrelic.noticeError())